Resources/Prototypes/_Sunrise/Entities/Mobs/Customization/Reptilian.yml

```
@@ -0,0 +1,482 @@
+
+- type: marking
+  id: ReptilianHornsLittleTrike
+  name: Унатх, рожки (Маленький рог)
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  speciesRestriction: [Reptilian]
+  sprites:
+  - sprite: _Sunrise/Mobs/Customization/horns.rsi
+    state: little_trike
+
+#Tails
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)